.github/workflows/update-providers-test.yml

```diff
@@ -0,0 +1,90 @@
+name: Test the bridge by previewing provider bridge upgrades
+
+on:
+
+  # These checks can be triggerred manually from the Actions tab, which already lets you specify
+  # which branch of the bridge to use for testing.
+  workflow_dispatch:
+    inputs: {}
+
+  # Alternatively, they can be triggered by opening a feature-* branch. Not having these enabled on
+  # normal branches and PRs is intentional as the checks consume a lot of resources.
+  push:
+    branches:
+    - feature-**
+    paths-ignore:
+    - CHANGELOG.md
+    tags-ignore:
+    - "*"
+    - "**"
```

Comment:
It seems pretty aggressive to trigger on push, especially if we want to turn on the rest of the providers. Should we restrict it to manual triggers?

Reply:
It's limited to branches called `feature-*`.

I think I've sent these used elsewhere in Pulumi.

So if we're building a feature that requires extra scrutiny, we start a feature-xyz branch and PRs based on that branch. When it merges it runs all the checks and reports back. That was the idea.

I'm happy to start with just manual triggers though, the policy here is up for debate.

Reply:
One thing to note is that there's no built-in monitoring currently. When you trigger this you don't know where to look for results necessarily. The feature-* branch has this slightly curious solution where PR descriptions cross-link back to the originating PR, that's serve as a poor-man's monitoring. I'd be curious if the works for manual triggers off PR or not. Can try that out

Reply:
We can try this for now, but I don't think an on-push trigger will scale to 50 PRs.